tool/speed.cc

```diff
@@ -2159,7 +2165,7 @@ bool Speed(const std::vector<std::string> &args) {
      ) {
     return false;
   }
-#if defined(BORINGSSL_FIPS)
+#if defined(AWSLC_FIPS)
```

Comment:
This raises an interesting point, BORINGSSL_FIPS and AWSLC_FIPS are only defined when building our libcrypto (we define BORINGSSL_FIPS in our CMakeList.txt). So customer's code building with AWS-LC's libcrypto can't tell at compile time if it is FIPS or not. This is creating an interesting issue: speed.cc isn't running these two tests if you build a separate libcrypto that is FIPS and then pass it in AWSLC_INSTALL_DIR or BORINGSSL_INSTALL_DIR.

Reply:
Yeah when this was added, we didn't really have benchmarking different FIPS libcryptos in mind. We should look into adding support for this some time down the line.